src/main/java/frc/robot/commands/ShooterTeleop.java

```diff
@@ -0,0 +1,37 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.XboxController;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.ShooterSystem;
+
+public class ShooterTeleop extends Command {
```

Comment:
This command was only for testing. You can delete it.

Reply:
done